docs/docs/contributing/01-plugins.md

```diff
@@ -1,20 +1,23 @@
 ---
-sidebar_position: 12
-description: Using and Developing Plugins
+description: Using and Developing plugins
 ---
 
-# Using and Developing Plugins
+# Developing plugins
 
 ## Using Plugins
 
-Ignite plugins offer a way to extend the functionality of the Ignite CLI. There are two core concepts within plugins : `Commands` and `Hooks`. Where `Commands` extend the cli's functionality, and `Hooks` extend existing command functionality.
+Ignite plugins offer a way to extend the functionality of the Ignite CLI. There
+are two core concepts within plugins : `Commands` and `Hooks`. Where `Commands`
+extend the cli's functionality, and `Hooks` extend existing command
+functionality.
 
-Plugins are registered in an Ignite scaffolded Blockchain project through the `config.yml`.
+Plugins are registered in an Ignite scaffolded Blockchain project through the
+`config.yml`.
 
 ### Adding plugins to a project
 
-Plugins are registered per project, using the `config.yml` file. To use a
-plugin within your project, add a `plugins` section with the following:
+Plugins are registered per project, using the `config.yml` file. To use a plugin
+within your project, add a `plugins` section with the following:
```

Comment:
This is all outdated now due to the global plugin config system.  We can merge this PR now and edit later, or bring it up to date here.

Wdyt?

Reply:
Let's do it in a separate PR. PRs like this one is mostly about moving things around.